packages/vite-node/src/utils.ts

```diff
@@ -45,14 +45,16 @@ export function isPrimitive(v: any) {
   return v !== Object(v)
 }
 
-export function toFilePath(id: string, root: string): string {
+export function toFilePath(id: string, root: string, raw = false): string {
```

Comment:
This became very confusing. Why as a developer I should use `raw` here? This function should just work, I already gave it path